openwisp_radius/api/views.py

```diff
@@ -359,6 +359,8 @@ def post(self, request, *args, **kwargs):
                     organization=Organization.objects.get(pk=self.request.auth),
                 )
                 batch = RadiusBatch(**options)
+                batch.full_clean()
```

Comment:
RadiusBatchSerializer extends ModelSerializer from openwisp-utils, which calls the full_clean during validation, so I suspect this is redundant.